lib/katello_deploy/installer.rb

```diff
@@ -41,12 +41,31 @@ def install_packages(packages)
 
     def setup_config
       return unless local_katello
+
       curr_dir = Dir.pwd
-      Dir.chdir(foreman_config_root) do
-        symlink("#{curr_dir}/katello-installer/config/katello.yaml", 'katello.yaml')
-        symlink("#{curr_dir}/katello-installer/config/katello-answers.yaml", 'katello-answers.yaml')
-        symlink("#{curr_dir}/katello-installer/config/katello.migrations", 'katello.migrations')
-      end
+
+      symlink(
+        "#{curr_dir}/katello-installer/config/katello-answers.yaml",
+        "#{foreman_config_root}/katello-answers.yaml"
+      )
+      symlink(
+        "#{curr_dir}/katello-installer/config/katello.migrations",
+        "#{foreman_config_root}/katello.migrations"
+      )
+
+      symlink(
+        "#{curr_dir}/katello-installer/config/katello-devel-answers.yaml",
+        "#{foreman_config_root}/katello-devel-answers.yaml"
+      )
+      symlink(
+        "#{curr_dir}/katello-installer/config/katello-devel.migrations",
+        "#{foreman_config_root}/katello-devel.migrations"
+      )
+
+      symlink(
+        "#{curr_dir}/katello-installer/_build/modules",
```

Comment:
You can do it the old way, however, the Rake task prunes the set of modules now to be a subset removing those found int he foreman-installer to avoid weird issues that we saw arise if two versions of the same module happen to exist in the modules path.